app/src/main/java/org/jellyfin/androidtv/integration/LeanbackChannelWorker.kt

```diff
@@ -192,10 +201,9 @@ class LeanbackChannelWorker(
 			item.canResume -> {
 				setWatchNextType(WatchNextPrograms.WATCH_NEXT_TYPE_CONTINUE)
 				setLastPlaybackPositionMillis((item.resumePositionTicks / TICKS_IN_MILLISECOND).toInt())
-			}
-			// Episode runtime has been determined
-			item.runTimeTicks != null -> {
-				setDurationMillis((item.runTimeTicks / TICKS_IN_MILLISECOND).toInt())
+				if (item.runTimeTicks != null) {
+					setDurationMillis((item.runTimeTicks / TICKS_IN_MILLISECOND).toInt())
+				}
```

Comment:
I'm not 100% sure if it's used for the NEW and NEXT types but maybe this should be moved outside of the `when {}` statement,